src/diff.test.ts

```diff
@@ -49,7 +49,7 @@ test('empty section', () => {
         'bundles',
       )} got bigger`,
       isEmpty: true,
-      children: renderBiggerTable({ assets: [] }),
```

Comment:
**nit** Is this a formatting / prettier thing?

Reply:
forgot to run prettier